### ISSUE TITLE
capture no `-k` option gracefully in `crypto.py`

### ISSUE THREAD

**Issue by lukehinds** [keylime/python-keylime#90]
capture no `-k` option gracefully in `crypto.py`
```
1553171329.0 - keylime.tenant - ERROR - You must specify one of -k, -f, or --cert to specify the key/contents to be securely delivered to the node
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/keylime-1.2-py2.7.egg/keylime/tenant.py", line 731, in main
    mytenant.preloop()
  File "/usr/lib/python2.7/site-packages/keylime-1.2-py2.7.egg/keylime/tenant.py", line 357, in preloop
    self.auth_tag = crypto.do_hmac(self.K,self.node_uuid)
  File "/usr/lib/python2.7/site-packages/keylime-1.2-py2.7.egg/keylime/crypto.py", line 87, in do_hmac
    h = HMAC.new(key,str(value),digestmod=SHA384.new())
  File "/usr/lib64/python2.7/site-packages/Cryptodome/Hash/HMAC.py", line 213, in new
    return HMAC(key, msg, digestmod)
  File "/usr/lib64/python2.7/site-packages/Cryptodome/Hash/HMAC.py", line 71, in __init__
    if len(key) <= digestmod.block_size:
TypeError: object of type 'NoneType' has no len()
```

**Comment by nabilschear** [comment]
i think this is actually a problem in `tenant.py` rather than `crypto.py`.  what is the command line you used do create it?  it should raise an Exception and quite trying to do other stuff.  if this is on the branch where you've removed the raise Exceptions, then, that would make sense.  It's just an example of a downstream problem like i referred to in https://github.com/keylime/python-keylime/pull/91#issuecomment-475299021

**Comment by lukehinds** [comment]
good point, let me park this issue for now while work on #91 - this could be a duplicate.

**Comment by lukehinds** [comment]
closing as resolved in #91
